tests/linearizability/linearizability_test.go

```diff
@@ -196,6 +204,18 @@ type trafficConfig struct {
 	traffic     Traffic
 }
 
+func validateEventsMatch(t *testing.T, ops [][]watchEvent) {
+	for i := 1; i < len(ops); i++ {
```

Comment:
Please add a comment describing when we expect different lengths... (one node off/partitioned during the test).

Reply:
Done